qprofiler2/src/org/qcmg/qprofiler2/Summarizer.java

```diff
@@ -8,7 +8,7 @@
  */
 package org.qcmg.qprofiler2;
 
-public interface Summarizer {
+public interface Summarizer { 
```

Comment:
same in this file (maybe all files?) - why is there a whitespace after all the opening block parentheses?